docs/GettingStarted/configuration/PelorusExporters.md

```diff
@@ -1,175 +1,326 @@
+                - example-secret
+        [...]
+        ```
+        to the exporter configuration.
+
+    1. To create a secret named **other-secret** in **pelorus** namespace, with the
+
+        - option **SERVER** with value **server_url**
+        - option **API_USER** with value **username**
+        - option **TOKEN** with value **token_value**
+
+        run
+        ```
+        oc create secret generic other-secret -n pelorus \
+        --from-literal=SERVER=server_url \
+        --from-literal=API_USER=username \
+        --from-literal=TOKEN=token_value
+        ```
+        Then, add
+        ```yaml
+        [...]
+                env_from_secrets:
+                - other-secret
+        [...]
+        ```
+        to the exporter configuration.
+
+### env_from_configmaps
+
+- **Required:** no
+- **Type:** list
+
+    List of ConfigMaps, like in the following example.
+    ```yaml
     env_from_configmaps:
-    - pelorus-config
-    - comittime-enterprise-config
-```
+    - example-configmap
+    ```
 
-### ConfigMap configuration values
+    Check the list of all available options per exporter type:
 
-Configuration for each exporter is done via ConfigMap objects. Best practice is to store the folder outside of local Pelorus Git repository and modify accordingly.
-Each ConfigMap must be in a separate file and must be applied to the cluster before deploying pelorus helm chart.
+    - [Deploy time](ExporterDeploytime.md#deploy-time-exporter-configuration-options)
+    - [Commit time](ExporterCommittime.md#commit-time-exporter-configuration-options)
+    - [Failure](ExporterFailure.md#failure-time-exporter-configuration-options)
 
-ConfigMap can be applied individually or all together:
-```shell
-# Only deploytime ConfigMap
-oc apply -f charts/pelorus/configmaps/deploytime.yaml
+    **ConfigMap Example**
 
-# All at once
-oc apply -f charts/pelorus/configmaps/
-```
+    To create a ConfigMap named **example-config** in **pelorus** namespace, with the
 
-Example ConfigMap for the `deploytime-exporter` with the unique name `deploytime-config`:
-```
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: deploytime-config
-  namespace: pelorus
-data:
-  PROD_LABEL: "default"    # "" / PROD_LABEL is ignored if NAMESPACES are provided
-  NAMESPACES: "default"    # ""
-```
-### Authentication to Remote Services
+    - option **APP_LABEL** with value **example**
+    - option **NAMESPACES** with value **one,two**
 
-Pelorus exporters make use of `personal access tokens` when authentication is
-required.  It is recommended to configure the Pelorus exporters with authentication
-via the `TOKEN` key to avoid connection rate limiting and access restrictions.
+    using the file **config.yaml**
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: example-config
+      namespace: pelorus
+    data:
+      APP_LABEL: "example"
+      NAMESPACES: "one,two"
+    ```
+    run
+    ```
+    oc apply -f config.yaml
```

Comment:
Missing namespace, this will apply to the current `oc project` if namespace is not specified.

Reply:
I will add namespace

Reply:
I tested without the namespace and it worked the same way (because of namespace in metadata?). Do you still want to add it?